doc/build-unix.md

```diff
@@ -69,7 +69,7 @@ Build requirements:
 
 Options when installing required Boost library files:
 
-1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+1. On at least Ubuntu 14.04+ ( > kernel 3.13 )  and Debian 7+( > kernel v3.2.41)  there are generic names for the
```

Comment:
Yes, **the document** is general for unix, **except** sections that refer to specific OSes/distributions, where specified as such.